sycl/test-e2e/OptionalKernelFeatures/throw-exception-for-out-of-registers-on-kernel-launch.cpp

```diff
@@ -2,6 +2,7 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 // XFAIL: *
+// XFAIL-TRACKER: CMPLRTST-24904
```

Comment:
Isn't it https://github.com/intel/llvm/issues/12679 ?

Reply:
Yes, I forgot to do a search for this.